tools/emptyDropsWrapper/README.md

```diff
@@ -0,0 +1,23 @@
+# emptyDropsWrapper
+
+Reads in an rds file containing the count matrix in dgCMatrix format in gene x droplet orientation and produces a CSV file indicating cell filtering and other cell metadat as returned by the emptyDrops function.
+
+## Requirements
+
+This script requires the following R packages to be installed: 
+
+* 'optparse'
```

Comment:
Minor nit-pick: I think you can remove the quotation marks around the packages now that they are formatted as a list instead of part of a sentence.

Reply:
done